test/client/nodes_test.rb

```diff
@@ -41,7 +41,7 @@
   it "gets the hot threads for the node(s)" do
     str = $client.nodes.hot_threads :read_timeout => 2
     assert_instance_of String, str
-    assert_match %r/cpu usage by thread/, str
+    assert_match %r/:::/, str
```

Comment:
@TwP: Yeah that's weird, and the docs for 1.4.0 don't have that param: https://www.elastic.co/guide/en/elasticsearch/reference/1.4/cluster-nodes-hot-threads.html#cluster-nodes-hot-threads

How about just a check for HTTP OK?